docs/htdocs/info/docs/api/variation/variation_tutorial.html

```diff
@@ -17,92 +18,17 @@ <h2 id="intro" style="font-size:large">Introduction</h2>
+<a href="/info/docs/api/core_tutorial/index.html">Ensembl Core API</a> and of the
+<a href="/info/docs/api/general_instructions.html">coding conventions</a> 
+used in the Ensembl APIs is assumed.
 </p>
-
-<br />
-<h2 id="connect">Connecting to a Variation DB</h2>
-
 <p>
-Connecting to an Ensembl Variation database is made simple by using the Bio::EnsEMBL::Registry module:
+Documentation about the Variation database schema is available <a href="variation_schema.html">here</a>, 
+and while not necessary for this tutorial, an understanding of the database tables may help as many of the adaptor modules are table-specific.
 </p>
 
-<pre class="code sh_perl">
-use Bio::EnsEMBL::Registry;
-
-my $registry = 'Bio::EnsEMBL::Registry';
-
-$registry-&gt;load_registry_from_db(
-  -host =&gt; 'ensembldb.ensembl.org',
-  -user =&gt; 'anonymous',
-);
-</pre>
-
-<div style="background-color:#F0F0F0;border:1px solid #CCC;padding:5px;margin-bottom:20px">
-  <h2 id="grch37" style="color:red">Connecting to GRCh37</h2>
-  <p>
-  If you are interested in data on the human <b>GRCh37</b> assembly you need to set the port number to <span style="color:red;font-weight:bold">3337</span>:
-  </p>
-
-<pre class="code sh_perl" style="margin:2px 5px 5px">
-use Bio::EnsEMBL::Registry;
-
-my $registry = 'Bio::EnsEMBL::Registry';
-
-$registry-&gt;load_registry_from_db(
-  -host =&gt; 'ensembldb.ensembl.org',
-  -user =&gt; 'anonymous',
-  -port =&gt; 3337,
-);
-</pre>
-  
-</div>
-
-<p> The use of the registry ensures you will load the correct versions of the
-Ensembl databases for the software release it can find on a database instance.
-Using the registry object, you can then create any of number of database
-adaptors. Each of these adaptors is responsible for generating an object of one
-type. The Ensembl Variation API uses a number of object types that relate to the
-data stored in the database. For example, in order to generate Variation
-objects, you should first create a Variation adaptor: </p>
-
-<pre class="code sh_perl">
-use Bio::EnsEMBL::Registry;
-
-my $registry = 'Bio::EnsEMBL::Registry';
-
-$registry-&gt;load_registry_from_db(
-  -host =&gt; 'ensembldb.ensembl.org',
-  -user =&gt; 'anonymous',
-);
-
-my $variation_adaptor = $registry-&gt;get_adaptor(
-  'human',     # species
-  'variation', # database type
-  'variation'  # object type
-);
-
-my $variation = $variation_adaptor-&gt;fetch_by_name('rs1333049');
-</pre>
-
 <p>
-	The <b>get_adaptor</b> method will automatically create a connection to the
-	relevant database; in the example above, a connection will be made to the
-	Variation database for human. The three parameters passed specify the
-	species, database and object type you require. Below is a non exhaustive
-	list of Ensembl Variation adaptors that are most often used
+Below is a non exhaustive list of Ensembl Variation adaptors that are most often used:
```

Comment:
Can we remove ReadCoverage from here. I am confident it isn't frequently used!

Reply:
I was thinking it didn't look popular!